lib/rokt_sdk.dart

```diff
@@ -114,3 +117,31 @@ class RoktSdk {
     await RoktSdkController.instance.setLoggingEnabled(enable: enable);
   }
 }
+
+/// Configuration settings for the Rokt SDK <br>
+///
+/// - Attributes
+///   - [ColorMode]? colorMode: preferred device color mode configuration
+@immutable
+class RoktConfig {
+  /// The device color mode your application is using
+  final ColorMode colorMode;
+
+  /// Constructor
+  ///
+  /// - Parameters
+  ///   - [ColorMode]? colorMode: preferred device color mode configuration
+  const RoktConfig({this.colorMode = ColorMode.system});
+}
+
+/// Enum representing device color modes
+enum ColorMode {
+  /// Request Light mode configuration
+  light,
```

Comment:
https://dart.dev/effective-dart/style#prefer-using-lowercamelcase-for-constant-names

Reply:
Same as in ReactNative, can it be part of RoktConfig ?

Reply:
![image](https://github.com/ROKT/rokt-sdk-flutter/assets/122851605/0b2a2b86-aabc-4491-a0c9-aa321993121d)
:( (think dart doesn't support inner classes at all)